docs/sources/developers/plugins/metadata.md

```diff
@@ -9,28 +9,128 @@ aliases = ["/docs/grafana/latest/plugins/developing/plugin.json/"]
+| `description` | string            | false    | Description of plugin. Used on the plugins page in Grafana and for search on grafana.com.                                     |
+| `keywords`    | array             | false    | Array of plugin keywords. Used for search on grafana.com.                                                                     |
+| `links`       | array             | false    | An array of link objects to be displayed on this plugin's project page in the form `{name: 'foo', url: 'http://example.com'}` |
+| `screenshots` | array             | false    | An array of screenshot objects in the form `{name: 'bar', path: 'img/screenshot.png'}`                                        |
+
+### author
+
+Information about the plugin author.
+
+#### Properties
+
+| Property | Type   | Required | Description               |
+|----------|--------|----------|---------------------------|
+| `name`   | string | **true** | Author's name.            |
+| `url`    | string | **true** | Link to author's website. |
+
+### logos
+
+SVG images that are used as plugin icons.
+
+#### Properties
+
+| Property | Type   | Required | Description                                                                                                                  |
+|----------|--------|----------|------------------------------------------------------------------------------------------------------------------------------|
+| `large`  | string | **true** | Link to the "large" version of the plugin logo, which must be an SVG image. "Large" and "small" logos can be the same image. |
+| `small`  | string | **true** | Link to the "small" version of the plugin logo, which must be an SVG image. "Large" and "small" logos can be the same image. |
+
+## queryOptions
+
+For data source plugins. There is a query options section in the plugin's query editor and these options can be turned on if needed.
+
+### Properties
+
+| Property        | Type    | Required | Description                                                                                                                |
+|-----------------|---------|----------|----------------------------------------------------------------------------------------------------------------------------|
+| `cacheTimeout`  | boolean | false    | For data source plugins. If the `cache timeout` option should be shown in the query options section in the query editor.   |
+| `maxDataPoints` | boolean | false    | For data source plugins. If the `max data points` option should be shown in the query options section in the query editor. |
+| `minInterval`   | boolean | false    | For data source plugins. If the `min interval` option should be shown in the query options section in the query editor.    |
+
+## routes
+
+For data source plugins. Proxy routes used for plugin authentication and adding headers to HTTP requests made by the plugin.
```

Comment:
Should link to https://grafana.com/docs/grafana/latest/developers/plugins/authentication/